src/modules/fetchcomment.js

```diff
@@ -0,0 +1,15 @@
+const commentsload = document.querySelector('.commentsload');
+
+export default async function fetchcomment(id, key) {
+  commentsload.innerHTML = '';
+  await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${key}/comments?item_id=${id}`)
+    .then((res) => res.json())
+    .then((output) => {
+      if (output.error) {
+        return;
+      }
+      output.forEach((comment) => {
+        commentsload.innerHTML += `<p><span>${comment.creation_date} ${comment.username}: ${comment.comment}</span></p>`;
+      });
```

Comment:
- [ ] Same here, you should Use `async` and `await` with the `fetch` method to fetch but I can see that you used Promises. Kindly avoid this

- Syntax: Follow this syntax which we could use in order to execute an asynchronous method async-await:

```
const getScores = async (id) => {
  const response = await fetch(
    `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${id}/scores`,
    {
      method: 'GET',
      headers: { Accept: 'application/json' },
    },
  );
  const data = await response.json();
  return data.result;
};
``` 
- For more documentation, you can check  [Promises vs Async Await](https://www.loginradius.com/blog/engineering/callback-vs-promises-vs-async-await/)

Reply:
The promises are part of the project requirement as mentioned here:
<img width="448" alt="callbacks" src="https://user-images.githubusercontent.com/77294268/212458244-bbb7c628-c9fb-4239-88d3-ffdcfd5abfdd.png">